tests/chocolatey-tests/features/ChocolateyCache.Tests.ps1

```diff
@@ -0,0 +1,40 @@
+﻿Describe 'Ensuring nupkgs cleared from temporary cache location (<Command>)' -ForEach @(
+    @{ Command = 'install' ; ExpectedExit = 0}
+    @{ Command = 'upgrade' ; ExpectedExit = -1}

```

Comment:
Why is this -1?

Reply:
This was because initially I had specific packages here, and `upgradepackage` has an exit of `-1` due to throwing in the beforeModify script. That's not there anymore, so this exit should be 0 (although that's not the failure, I think the failure in Test Kitchen was related to not restoring the snapshot 🤔 )

Reply:
We can chat about this more when you come online properly, as I am assuming that this isn't you started work 😄